### PR TITLE
#160609507 Set-up testing environment

### PR DESCRIPTION
**Pull request by mendozabree**
#### What does this PR do?

Setup testing environment for our codebase
#### Description of Task to be completed?

Currently, there is no testing set up and the test file is in the `src` directory.

This task creates a folder to hold all tests that will be written and adds enzyme that we will use for our tests.

#### How should this be manually tested?

- Run `npm install` to install all dependencies
- Run `npm run test tests`

#### What are the relevant pivotal tracker stories?

[#160609507](https://www.pivotaltracker.com/story/show/160609507)

#### Any background context you want to add?
N/A

#### Screenshots
screenshot 2018-10-30 at 15 46 20

### CODE REVIEW

**Review by RonKbS — APPROVED** [review]
LGTM

**Comment by kakaemma** [comment]
LGTM

**Review by kakaemma — APPROVED** [review]
LGTM

**Review by samuelbwambale — APPROVED** [review]
Test running okay. LGTM

**Review by MuhanguziDavid — APPROVED** [review]
LGTM

File: src/tests/App.test.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { shallow } from 'enzyme';
+import React from 'react';
+import App from '../App';
+
+it('renders without crashing', () => {
+  shallow(<App />);
```
Comment (dondrzzy) [inline src/tests/App.test.js]: Every test should `expect` something. For rendering something properly. You could use `toMatchSnapshot`. Can get more context from [here](https://stackoverflow.com/questions/38130214/how-to-test-arrow-function-in-react-es6-class/51944117#51944117).

**Review by dondrzzy — COMMENTED** [review]
Well done @mendozabree. Please refactor.

File: src/tests/App.test.js (hunk 1 above)
Comment (mendozabree) [reply src/tests/App.test.js]: Thanks, let me work on that.

**Pull request merged**